Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.4.2
+
+### Install godep, dependency manager for golang ###
+RUN go get github.com/tools/godep
+RUN go get -u github.com/golang/lint/golint
+
+### Set up working directory ###
+RUN mkdir -p /go/src/github.com/brainly/pencil-go
+WORKDIR /go/src/github.com/brainly/pencil-go
+ADD . /go/src/github.com/brainly/pencil-go
+
+### Build the project including dependencies ###
+RUN godep go install
```

Comment:
As I remember we agreed that the project should already contain Godeps. Should we make this decision consistent?